### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by Chris-Hibbert**

### CODE REVIEW

**Comment by Chris-Hibbert** [comment]
This is just the doc updates, which we should be able to handle separately.

**Comment by Chris-Hibbert** [comment]
I intended this to be a separate PR from https://github.com/Agoric/SwingSet/pull/148/files, but managed to fat-finger that. 

You commented on these doc updates there, and I've made those changes here. I'll remove these commits from that PR, since it doesn't depend on them.

File: docs/devices.md [hunk 1]
```diff
@@ -152,18 +144,71 @@ The `liveSlots` helper for constructing Vats provides both an `E()` wrapper
...
+### Device API
+
+Devices are very much like Vats, and they way they are constructed reflects
+that. The factory function for each is called with `syscall` object that
+they can use to talk to the kernel, and they return a `dispatch` object
+with which the kernel can invoke them. The specific methods on `syscall`
+and `dispatch` are different for devices:
+
+* `syscall.sendOnly(targetSlot, method, args) -> undefined`
+* `dispatch.invoke(deviceID, method, args) -> { results }`
```
Comment (warner) [inline docs/devices.md]: the return value is CapData, rather than an object with a `.results` property that contains capdata, so I'd remove the curlies

File: src/devices/mailbox.js [hunk 2]
```diff
@@ -166,6 +162,8 @@ export function buildMailbox(state) {
     }
   }
 
+  // Functions made available to the host: these are used for inbound
```
Comment (warner) [inline src/devices/mailbox.js]: I'd either move this back up to cover `deliverInbound`, or reword it to "`deliverInbound` is made available to the host, for it to deliver inbound messages..". I don't want to give the impression that `srcPath` and `endowments` are made available to the host (well, they are, but it only uses them at config time, and passes them directly into the controller/kernel to build the device, whereas `deliverInbound` is used at runtime, every time a new message arrives)

File: docs/devices.md (hunk 1 above)
Comment (Chris-Hibbert) [reply docs/devices.md]: done

File: src/devices/mailbox.js (hunk 2 above)
Comment (Chris-Hibbert) [reply src/devices/mailbox.js]: done
